editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/ReferenceCodeForm.tsx

```diff
@@ -16,8 +16,8 @@ export default function ReferenceCodeForm({
 }: FormProps) {
   const formSchema = Yup.object().shape({
     referenceCode: Yup.string()
-      .min(3, "Code must be 3 characters long")
-      .max(3, "Code must be 3 characters long")
+      .min(3, "Code must be at least 3 characters long")
+      .max(5, "Code cannot exceed 5 characters long")
```

Comment:
All our councils so far have had 3 digit reference codes, but this isn't true for all PD "organisations". "Combined authorities" especially are likely to have longer ones eg https://submit.planning.data.gov.uk/organisations/local-authority:YNYCA

I don't anticipate we'll ever support "National Park Authorities" so can ignore these even longer ones eg https://submit.planning.data.gov.uk/organisations/national-park-authority:Q5225646